Src/BootCamp.Chapter/Students/Student.cs

```diff
@@ -0,0 +1,27 @@
+﻿using BootCamp.Chapter.Teachers;
+using System;
+using System.Collections.Generic;
```

Comment:
Codacy found an issue: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5789960)